Clients/GUI/Shared/NavMenu.razor

```diff
@@ -52,6 +53,7 @@
 
     private void SignOut()
     {
+        LocalStorage.ClearAsync();
```

Comment:
Local Storage is different object for each user?

Reply:
> Local Storage is different object for each user?

Yes it is, but that's to make sure that there's one user's info only
Idk, mb there's a better solution, then suggest changes